python/lsst/analysis/drp/dataSelectors.py

```diff
@@ -12,9 +12,9 @@ class FlagSelector(DataFrameAction):
     selectWhenFalse = ListField(doc="Names of the flag columns to select on when False",
                                 dtype=str,
                                 optional=False,
-                                default=["base_PixelFlags_flag"])
+                                default=[])
 
-    selectWhenTrue = ListField(doc="Names of the flag columns to select on when False",
+    selectWhenTrue = ListField(doc="Names of the flag columns to select on when True",
```

Comment:
Was this just wrong before, or did you change the functionality?

Reply:
It was just wrong. Over enthusiastic copy and paste.